olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/Types.h

```diff
@@ -78,6 +82,12 @@ using PrefetchTilesResponse = Response<PrefetchTilesResult>;
 /// Prefetch completion callback type
 using PrefetchTilesResponseCallback = Callback<PrefetchTilesResult>;
 
+/// Subscribe response type
+using SubscribeApiResponse = Response<model::SubscribeResponse>;
```

Comment:
Sorry, I missed seeing this. These are private responses for now so please move them to StreamApi but still you can use the Response<> template.